### PR TITLE
Fix small toolbar sizing, some documentation and example styles

### PR DESCRIPTION
**Pull request by scotchfield**
A few small pieces of feedback from design. The padding on small toolbars was expanding the size beyond the expected height. The other details relate to examples in the documentation.

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 7adc8ebdabdd1802cc84798b1068e1c8c708d002

https://deploy-preview-235--wonder-blocks.netlify.com

File: packages/wonder-blocks-toolbar/generated-snapshot.test.js [hunk 1]
```diff
@@ -126,8 +142,6 @@ describe("wonder-blocks-toolbar", () => {
         const example = (
             <View style={style}>
                 <Toolbar color="dark" title="Title" />
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                eiusmod tempor incididunt ut labore et dolore magna aliqua.
```
Comment (kevinbarabash) [inline packages/wonder-blocks-toolbar/generated-snapshot.test.js]: Sorry for not picking this up.  I keep forgetting that design that doesn't want anything in here.  It feels weird to have all this blank space.  🤷‍♂️

Comment (scotchfield) [reply packages/wonder-blocks-toolbar/generated-snapshot.test.js]: No problem at all! I should have caught it earlier.

File: packages/wonder-blocks-toolbar/components/toolbar.md [hunk 2]
```diff
@@ -60,10 +73,15 @@ const tryAgain = <Button kind="secondary">Try again</Button>;
 const {default: Icon, icons} = require("@khanacademy/wonder-blocks-icon");
 const IconButton = require("@khanacademy/wonder-blocks-icon-button").default;
 const Link = require("@khanacademy/wonder-blocks-link").default;
+const {LabelLarge} = require("@khanacademy/wonder-blocks-typography");
 
 const closeButton = <IconButton icon={icons.dismiss} kind="tertiary" style={{ width: 28 }} />;
```
Comment (kevinbarabash) [inline packages/wonder-blocks-toolbar/components/toolbar.md]: I think the style here should be `style={{marginLeft: -12}}`

Comment (scotchfield) [reply packages/wonder-blocks-toolbar/components/toolbar.md]: Sure, sounds good. I wasn't sure about the focus ring brushing up against the border like that when testing before.

**Comment by scotchfield** [comment]
Thanks! Just waiting for final confirmation from design and then we can merge this.

**Comment by tatianasalazar** [comment]
sorry! we are good to go. Thank you Scott!

**Comment by kevinbarabash** [comment]
@scotchfield this needs rebasing before it can be merged into `master`.

**Pull request merged**